bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/factory/OpenWeatherMapHandlerFactory.java

```diff
@@ -100,8 +101,12 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
             return new OpenWeatherMapWeatherAndForecastHandler(thing, timeZoneProvider);
         } else if (THING_TYPE_UVINDEX.equals(thingTypeUID)) {
             return new OpenWeatherMapUVIndexHandler(thing, timeZoneProvider);
+        } else if (THING_TYPE_ONECALL_WEATHER_AND_FORECAST.equals(thingTypeUID)) {
+            return new OpenWeatherMapOneCallHandler(thing, timeZoneProvider);
+        } else if (THING_TYPE_ONECALL_HISTORY.equals(thingTypeUID)) {
+            return new OpenWeatherMapOneCallHistoryHandler(thing, timeZoneProvider);
         }
-
+        logger.error("Could not create handler for thing {}, UID {}",thing,thingTypeUID);
```

Comment:
Remove this log

Reply:
Done (although I consider it helpful :-) )